### ISSUE TITLE
Synchronize setting channels with stored values with the Csound performance

### ISSUE THREAD

**Issue by gogins** [gogins/webkit-opcodes#12]
Currently, the WebKit opcodes open a Web page asynchronously. Often, JavaScript code on that page has the function of setting controls on the page to stored values, and these controls then update Csound global variables.

As a result, the desired values show up in the control variables only after performance has been going on for some time. Typically there is a lag of 5 to 7 seconds between the start of the Csound performance and the updating of channels and variables with the intended values. This causes big problems at the start of most pieces. It's even more of a problem for off-line rendering, of course.

Conceptually this problem has two possible solutions:

- [x] Load Web pages synchronously. In this case the Csound performance will pause or block until the Web page has completely loaded. _I think this would be way too much of a hack._
- [x] Store the control values only in Csound global variables. The Web page will update its controls from those. _This works in both real-time audio and off-line rendering_. It is necessary to first ensure that the Csound global variables have usable values, and of course the Csound variables and the HTML control ids have to be identical, as before.

It would be nice to have some automatic way of parsing the dat.gui settings JSON into Csound code. There will probably always be some hack or some manual step here.

**Comment by gogins** [comment]
Perhaps the best I can do is iterate over the persistent controls and parse the "id - value" pairs into "id init value" lines, and then get them in the clipboard.
